even_fib_nums.js

```diff
@@ -5,22 +5,47 @@
  * @param  {Number} maxFibValue
  * @return {Number} sum
  */
-function _sumFibs( maxFibValue ) {
-  var sum = 0;
+function _sumFibs(maxFibValue) {
+  var sum = 2;
+  let fibArray = [1, 2];
 
-  // do your work here
+  if (maxFibValue > 2) {
+    while (fibArray[fibArray.length - 1] <= maxFibValue) {
+      let length = fibArray.length;
+      let current = fibArray[length - 2] + fibArray[length - 1];
+      fibArray.push(current);
+
+      if ((current % 2 === 0) && (current <= maxFibValue)) {
+        sum += current;
+      }
+    }
+
+  } else {
+    return maxFibValue;
+  }
 
   return sum;
```

Comment:
This could return 2 if maxFibValue is the value 1.

Reply:
I think what I wanted you to understand is that if maxFibValue = 1, the return should be 0, but for maxFibValue = 2, the return is 2.